README.md

```diff
@@ -166,7 +166,20 @@ For other distros, you'll need to build and install from source:
     export PKG_CONFIG_PATH="/usr/local/opt/openssl/lib/pkgconfig:$PKG_CONFIG_PATH"
     ```
 
-2.  Build and install.
+2. Clone the repository
```

Comment:
Building from sources means building from source tarballs.

Those who build from Git repositories usually know how to cloen the repository.